### PR TITLE
Add ability to "drop" a break command.

### PR DESCRIPTION
**Pull request by real-yfprojects**
See sjurba#30

### CODE REVIEW

File: lib/reducer.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-const actions = ['pick', 'fixup', 'squash', 'reword', 'edit', 'drop'];
+const actions = ["pick", "fixup", "squash", "reword", "edit", "drop"];
 const [DOWN, UP] = [1, -1];
 
 function deepFreeze(obj) {
   var propNames = Object.getOwnPropertyNames(obj);
   propNames.forEach(function (name) {
     var prop = obj[name];
-    if (typeof prop === 'object' && prop !== null) {
+    if (typeof prop === "object" && prop !== null) {
```
Comment (perlun) [inline lib/reducer.js]: Thanks for the effort @real-yfprojects. There's a lot of noise in the PR like these (double-quote to single quote changes, added trailing comma etc). Could you edit it to minimize the diff? to more carefully try and separate functionality from style changes.

Comment (real-yfprojects) [reply lib/reducer.js]: I am sorry. I forgot about that side effect of running a code formatter.

**Comment by perlun** [comment]
Thanks @real-yfprojects! :+1: I tried this PR now, but unfortunately it still doesn't work. :thinking:

**Comment by real-yfprojects** [comment]
I managed to get the git repo running and fixed it. Feel free to tidy up any of the code if it misses the use of a cleaner javascript feature.

**Comment by perlun** [comment]
> I managed to get the git repo running and fixed it. Feel free to tidy up any of the code if it misses the use of a cleaner javascript feature.

Thanks! Now we're getting somewhere. :slightly_smiling_face: It does work, but I found two bugs when quickly testing it locally:

- The `drop`/delete command moves the cursor backward one step. That doesn't feel intuitive to me. I'd prefer it to stay on the current line.
- Selecting multiple lines (like 5) does not delete all the lines, only a few of them. :thinking: Three in my case.

Once we have these bugs sorted out, I see no problem with merging this PR into my branch. Thanks for the effort!

**Comment by real-yfprojects** [comment]
>  * The `drop`/delete command moves the cursor backward one step. That doesn't feel intuitive to me. I'd prefer it to stay on the current line.

That's on purpose because I thought it should revert `b` but I now think that your right. I'll change it.

> * Selecting multiple lines (like 5) does not delete all the lines, only a few of them. thinking Three in my case.

Fixed now.

**Comment by perlun** [comment]
Thanks! It's now working almost perfectly, only one bug left (that I could find): try pressing `b` and adding a line at the end, then press "drop" or the Delete key on this line. That leaves the cursor _after_ the last line. :slightly_smiling_face: 

So the logic should be:

- If not placed on the last line => stay
- If deleting the last line => move cursor one line up

**Comment by real-yfprojects** [comment]
Fixed it.

**Comment by perlun** [comment]
Excellent, thanks! Looks really good now. Thanks for the effort. :bow: Will merge this into my PR now.

File: index.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ const args = require('minimist')(process.argv, {
   }
 });
 
+console.log("LOG")
```
Comment (perlun) [inline index.js]: Hmm, this seems like some debug code left in. Can it be removed?

File: package.json [hunk 3]
```diff
@@ -37,9 +37,9 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
     "chai-sinon": "^2.8.1",
-    "coveralls": "^2.12.0",
+    "coveralls": "^3.1.1",
     "istanbul": "^0.4.5",
-    "mocha": "^3.2.0",
+    "mocha": "^9.2.2",
```
Comment (perlun) [inline package.json]: This change + the lockfile change should probably be reverted, to provide a cleaner diff for @sjurba to review in the target PR.

**Comment by perlun** [comment]
> Excellent, thanks! Looks really good now. Thanks for the effort. bow Will merge this into my PR now.

Noted some more minor things. Once they're fixed, I'll get this merged. Thanks!

**Comment by real-yfprojects** [comment]
> Noted some more minor things. Once they're fixed, I'll get this merged. Thanks!

Those slipped in during rebasing. I rebased again to fix them.

**Comment by perlun** [comment]
Thanks! Looks good & re-tested briefly, merging into my branch. :pray: :+1:

**Pull request merged**